tests/network-tests/run-runtime-upgrade-tests.sh

```diff
@@ -167,7 +167,8 @@ function main {
     # 4. early chain db init
     init_chain_db
     echo >&2 "chain db initialized"
-    # 5. start node
+    # 5. start node using new version
+    export JOYSTREAM_NODE_TAG=${TARGET_RUNTIME}
     CONTAINER_ID=$(start_joystream_node)
```

Comment:
The `CONTAINER_ID` variable is no longer used except for assignment here. I think it should be removed?